### PR TITLE
Task/27020 Create aircraft maintenance and repair requests

### PR DESCRIPTION
**Pull request by chinhhq-0603**
https://edu-redmine.sun-asterisk.vn/issues/27020

### CODE REVIEW

**Comment by chinhhq-0603** [comment]
Screencast from 21-08-2020 08_38_00

**Review by chienpt98 — APPROVED** [review]

File: app/views/schedules/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    <%= f.submit t(".submit"), class: "btn btn-primary" %>
+  <% end %>
+<% else %>
+  <h1><%= t(".change") %> <%= params[:plane_id] %> <%= t(".into_ready") %></h1>
+  <%= form_for @schedule do |f| %>
+    <%= f.label :plane_id, t(".plane_number") %>
+    <%= f.text_field :plane_id,:value => params[:plane_id], class: "form-control" %>
+    <%= f.hidden_field :user_id,:value => current_user.id %>
+    <%= f.label :location_id, t(".location") %>
+    <%= f.collection_select :location_id, Location.where(warehouse_id: Settings.warehouse.ready, filled: Settings.condition.false), :id, :id_initial, {prompt: true}, {class: "form-control", required: "true"}%>
```
Comment (hieplt-0914) [inline app/views/schedules/new.html.erb]: viết scope đi em

File: app/views/schedules/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<% if params[:status].eql? Settings.plane.status.ready %>
+  <h1><%= t(".change") %> <%= params[:plane_id] %> <%= t(".into_gara") %></h1>
+  <%= form_for @schedule do |f| %>
+    <%= f.label :plane_id, t(".plane_number") %>
+    <%= f.text_field :plane_id,:value => params[:plane_id], class: "form-control" %>
+    <%= f.hidden_field :user_id,:value => current_user.id %>
+    <%= f.label :location_id, t(".location") %>
+    <%= f.collection_select :location_id, Location.where(warehouse_id: Settings.warehouse.gara, filled: Settings.condition.false), :id, :id_initial, {prompt: true}, {class: "form-control", required: "true"}%>
```
Comment (hieplt-0914) [inline app/views/schedules/new.html.erb]: Viết scope đi em

File: app/views/schedules/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<% if params[:status].eql? Settings.plane.status.ready %>
+  <h1><%= t(".change") %> <%= params[:plane_id] %> <%= t(".into_gara") %></h1>
+  <%= form_for @schedule do |f| %>
+    <%= f.label :plane_id, t(".plane_number") %>
+    <%= f.text_field :plane_id,:value => params[:plane_id], class: "form-control" %>
```
Comment (hieplt-0914) [inline app/views/schedules/new.html.erb]: sử dụng value: params[:plane_id]

Comment (chinhhq-0603) [reply app/views/schedules/new.html.erb]: ok a

File: app/views/schedules/new.html.erb (hunk 2 above)
Comment (chinhhq-0603) [reply app/views/schedules/new.html.erb]: ok a

File: app/views/schedules/new.html.erb (hunk 1 above)
Comment (chinhhq-0603) [reply app/views/schedules/new.html.erb]: ok a

**Comment by chinhhq-0603** [comment]
Screencast from 21-08-2020 11_37_12

**Comment by chinhhq-0603** [comment]
ready

**Comment by chinhhq-0603** [comment]
Screenshot from 2020-08-21 13-05-13

**Comment by chinhhq-0603** [comment]
Screenshot from 2020-08-21 13-20-37

**Pull request merged**